db/migrate/20161214214540_email_encryption_cost.rb

```diff
@@ -0,0 +1,9 @@
+class EmailEncryptionCost < ActiveRecord::Migration
+  def up
+    add_column :users, :email_encryption_cost, :string, default: '4000$8$4$'
```

Comment:
this cost is different per environment, can we pull this from the env? or is this the current production values?